active/NavigationView/HierarchicalNav/HierarchicalNavigationView.md

```diff
@@ -0,0 +1,363 @@
+                Children = new ObservableCollection<Category>() {
+                    new Category(){
+                        Name = "Menu Item 6",
+                        Icon = "Icon",
+                        Children = new ObservableCollection<Category>() {
+                            new Category() { Name  = "Menu Item 7", Icon = "Icon" },
+                            new Category() { Name  = "Menu Item 8", Icon = "Icon" }
+                        }
+                    }
+                }
+            },
+            new Category(){ Name = "Menu Item 9", Icon = "Icon" }
+        };
+    }
+
+    private void OnItemInvoked(object sender, NavigationViewItemInvokedEventArgs e)
+    {
+        var clickedItem = e.InvokedItem;
+        var clickedItemContainer = e.InvokedItemContainer;
+    }
+
+    private void OnItemExpanding(object sender, NavigationViewItemExpandingEventArgs e)
+    {
+        var nvib = e.ExpandingItemContainer;
+        if(nvib != null)
+        {
+            var name = "Last Expanding: " + nvib.Content;
+            ExpandingItemLabel.Text = name;
+        }
+        else
+        {
+            ExpandingItemLabel.Text = "Last Expanding: ERROR - No container returned!";
+        }
+    }
+
+    private void OnItemCollapsed(object sender, NavigationViewCollapsedEventArgs e)
+    {
+        var nvib = e.CollapsedItemContainer;
+        if (nvib != null)
+        {
+            var name = "Last Collapsed: " + nvib.Content;
+            CollapsedItemLabel.Text = name;
+        }
+        else
+        {
+            CollapsedItemLabel.Text = "Last Collapsed: ERROR - No container returned!";
+        }
+    }
+}
+```
+
+## Selection
+By default, any item can contain children, be invoked, or be selected.
+There's more detail about the [ItemInvoked](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.iteminvoked?view=winui-2.3) and [SelectionChanged](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationview.selectionchanged?view=winui-2.3) events later in this document.
+When providing users with a hierarchical tree of navigation options, you may choose to make parent items non-selectable, for example when your app doesn't have a destination page associated with that parent item.
+To prevent an item from showing the selection indicator when invoked, set its [SelectsOnInvoked](https://docs.microsoft.com/en-us/uwp/api/microsoft.ui.xaml.controls.navigationviewitem.selectsoninvoked?view=winui-2.3) property to False.
+
+```xaml
+<DataTemplate x:Key="NavigationViewMenuItem" x:DataType="local:Category">
+    <muxc:NavigationViewItem Content="{x:Bind Name}" 
+        MenuItemsSource="{x:Bind Children}"
+        SelectsOnInvoked="{x:Bind IsLeaf}" />
+</DataTemplate>
+
+<muxc:NavigationView x:Name="navview" 
+    MenuItemsSource="{x:Bind categories, Mode=OneWay}" 
+    MenuItemTemplate="{StaticResource NavigationViewMenuItem}">
+   
+</muxcontrols:NavigationView>
```

Comment:
The XML namespace prefixes here don't appear to match up

Reply:
Recommend: fix (muxc vs muxcontrols)